src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/ProxyRoutes.scala

```diff
@@ -2,26 +2,36 @@ package org.broadinstitute.dsde.workbench.leonardo.api
 
 import akka.http.scaladsl.server.Directives._
 import akka.http.scaladsl.server.Route
+import com.typesafe.config.ConfigFactory
 import com.typesafe.scalalogging.LazyLogging
+import net.ceedubs.ficus.Ficus._
 import org.broadinstitute.dsde.workbench.leonardo.model.{ClusterName, GoogleProject}
 import org.broadinstitute.dsde.workbench.leonardo.service.ProxyService
+import scala.concurrent.ExecutionContext.Implicits.global
 
 /**
   * Created by rtitle on 8/4/17.
   */
 trait ProxyRoutes { self: LazyLogging =>
   val proxyService: ProxyService
   protected val tokenCookieName = "FCToken"
+  private val config = ConfigFactory.parseResources("leonardo.conf").withFallback(ConfigFactory.load())
+  private val whitelist = config.as[(List[String])]("whitelist")
```

Comment:
Again this could be initialized once in `Boot.scala` and passed to the *Routes classes which need it.